tests/row_generation/test_MethodPlaceNotationGenerator.py

```diff
@@ -56,5 +58,113 @@ def test_parse_block(self):
+            ("Grandsire Triples", 7),
+            ("Grandsire Caters", 9),
+            ("Grandsire Cinques", 11),
+        ]
+
+        for (method_title, expected_stage) in test_cases:
+            with self.subTest(method_title=method_title, expected_stage=expected_stage):
+                generator = generator_from_special_title(method_title)
+                self.assertIsInstance(generator, PlaceNotationGenerator)
+                self.assertTrue(expected_stage, generator.stage)
+
+    def test_grandsire__other_stages(self):
+        test_cases = [
+            ("Grandsire Singles"),
+            ("Grandsire Minimus"),
+            ("Grandsire Minor"),
+            ("Grandsire Major"),
+            ("Grandsire Royal"),
+            ("Grandsire Maximus"),
+        ]
+
+        for (method_title) in test_cases:
+            with self.subTest(method_title=method_title):
+                generator = generator_from_special_title(method_title)
+                self.assertIsNone(generator)
+
+    def test_grandsire__in_title(self):
+        test_cases = [
+            ("Reverse Grandsire Doubles"),
+            ("Double Grandsire Triples"),
+            ("Grandsire Five Little Bob Triples")
+        ]
+
+        for (method_title) in test_cases:
+            with self.subTest(method_title=method_title):
+                generator = generator_from_special_title(method_title)
+                self.assertIsNone(generator)
+
+    def test_stedman__odd_stages(self):
+        test_cases = [
+            ("Stedman Doubles", 5),
+            ("Stedman Triples", 7),
+            ("Stedman Caters", 9),
+            ("Stedman Cinques", 11),
+        ]
+
+        for (method_title, expected_stage) in test_cases:
+            with self.subTest(method_title=method_title, expected_stage=expected_stage):
+                generator = generator_from_special_title(method_title)
+                self.assertIsInstance(generator, PlaceNotationGenerator)
+                self.assertTrue(expected_stage, generator.stage)
+
+    def test_stedman__other_stages(self):
+        test_cases = [
+            ("Stedman Singles"),
+            ("Stedman Minimus"),
+            ("Stedman Minor"),
+            ("Stedman Major"),
+            ("Stedman Royal"),
+            ("Stedman Maximus"),
+        ]
+
+        for (method_title) in test_cases:
+            with self.subTest(method_title=method_title):
+                generator = generator_from_special_title(method_title)
+                self.assertIsNone(generator)
+
+    def test_plain_hunt__all_stages(self):
+        test_cases = [
+            ("Plain Hunt Singles", 3),
+            ("Plain Hunt Doubles", 5),
+            ("Plain Hunt Triples", 7),
+            ("Plain Hunt Caters", 9),
+            ("Plain Hunt Cinques", 11),
+            ("Plain Hunt Minimus", 4),
+            ("Plain Hunt Minor", 6),
+            ("Plain Hunt Major", 8),
+            ("Plain Hunt Royal", 10),
+            ("Plain Hunt Maximus", 12),
```

Comment:
I wonder if it would be good to support `Plain Hunt 8` or `Plain Hunt on 8`.  It's probably not a big deal, though...

Reply:
Done, though allowing `Grandisre 5` to slip through as well